### ISSUE TITLE
[BUG]: NT struct array topic doesn't publish schema

### ISSUE THREAD

**Issue by truher** [robotpy/mostrobotpy#60]
### Problem description

```
x = ntcore.NetworkTableInstance.getDefault().getStructArrayTopic("foo", Translation2d).publish()
x.set([Translation2d(0,0)])
```
is rendered as  opaque in glass, and there is only one publisher, "foo."

adding 
```
y = ntcore.NetworkTableInstance.getDefault().getStructTopic("foo", Translation2d).publish()
y.set(Translation2d(0,0)
```
adds the "/.schema" publisher and fixes the array rendering.

### Operating System

Linux

**Comment by virtuald** [comment]
WPILib fix in https://github.com/wpilibsuite/allwpilib/pull/6303

**Comment by auscompgeek** [comment]
Fixed in v2024.3.1
